dotcms-integration/src/test/java/com/dotcms/datagen/TemplateLayoutDataGen.java

```diff
@@ -5,18 +5,20 @@
 import com.dotmarketing.portlets.containers.model.FileAssetContainer;
 import com.dotmarketing.portlets.templates.design.bean.*;
 
-import javax.swing.*;
 import java.util.*;
 
+import static com.dotcms.util.CollectionsUtils.list;
+
 public class TemplateLayoutDataGen  {
 
     private List<TemplateLayoutRow> rows;
 
     private List<TemplateLayoutColumn> currentColumns;
 
-    private Map<String, List<String>> containersIds = new HashMap<>();
-    final Map<String, List<String>> containersIdsInSidebar = new HashMap<>();
+    private List<ContainerUUID> containersIds = new ArrayList<>();
+    final List<ContainerUUID> containersIdsInSidebar = new ArrayList<>();
```

Comment:
This can be private.

Reply:
done https://github.com/dotCMS/core/pull/29752/commits/b5a1836872a6ccdd83eac1ef97876df83d0f56d8